### ISSUE TITLE
Grayed out elements in Options, Audio properties and other miscellaneous functions

### ISSUE THREAD

**Issue by flandriai** [mpc-qt/mpc-qt#58]
- Opening the Options menu on the Player tab the complete "Title bar" panel is grayed out, and most parts of "Other" and "History" panels.
- Formats tab completely empty on Windows.
- Playback tab: There's a "Speed step" option, but there is no way to change playback speed.It would be great to have the same option like in MPC-HC or in SMPlayer (+ / - 10% speed step with option of using high pitched sound too. (Pitch correction is already added.)
- Miscellaneous panel grayed out.
- Ability to hide "Playlist" panel by default is missing.

However the **Frame step** function already works **much better** comparing to any other players, as:

- there's no sound (video automatically muted) when using frame step both forwards and backwards
- and the frame step function works **smoothly**, without any glich. Whether you step forward or **backward** the function operates excellently.

**Comment by cmdrkotori** [comment]
Yeah half the stuff in the settings window isn't implemented, but you see I never use those features myself so it'll get done when it gets done.

**Comment by cmdrkotori** [comment]
As of a recent-ish commit in git master, all/most of these are now hidden: the user will no longer know what they're missing. :^)
